content/index.de.md

```diff
@@ -0,0 +1,248 @@
+
+Das Web bietet für Menschen mit Behinderungen eine noch nie dagewesene Möglichkeit für den Zugang zu Informationen sowie für die Interaktion, in dem Sinne, dass Barrieren von Print-, Audio- und Videomedien viel einfacher mittels Webtechnologien beseitigt werden können.
+
+Barrierefreiheit unterstützt die soziale Inklusion von Menschen mit Behinderungen und anderen, darunter:
+
+- älteren Menschen
+- Menschen in ländlichen Gegenden
+- Menschen in Entwicklungsländern
+
+**Es gibt auch aus unternehmerischer Sicht ein starkes Argument für Barrierefreiheit.** Wie im vorigen Abschnitt gezeigt, verbessert barrierefreies Design die allgemeine Nutzer:innenerfahrung und Befriedigung, insbesondere in einer Vielzahl von Umständen, über verschiedene Geräte hinweg und für ältere Nutzer:innen. Barrierefreiheit kann Ihre Marke stärken, Innovation fördern und Ihre Marktreichweite erhöhen.
+
+Barrierefreiheit ist in vielen Fällen **gesetzlich verpflichtend**.
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Weitere Infos zur Bedeutung von Barrierefreiheit" class="simple aside" %}
+{:/}
+
+-   Allgemeine Informationen über die geschäftlichen Vorteile finden Sie unter [[Gründe für digitale Barrierefreiheit aus unternehmerisches Sicht]](/business-case/).
+    -   Beispiele für Vorteile von [barrierefreien Audio- und Videomedien](/media/av/) finden Sie unter [Vorteile für Unternehmen](/media/av/users-orgs/#benefits).
+-   Anleitungen für die Erfassung von rechtlichen Anforderungen finden Sie im archivierten [Rechtliche Faktoren](https://www.w3.org/WAI/business-case/archive/pol).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Das Web barrierefrei machen {#making}
+
+Barrierefreiheit im Web hängt vom Zusammenspiel zahlreicher Komponenten ab, unter anderem Webtechnologien, Webbrowsern und anderen \"User Agents\", Erstellungswerkzeugen und Webseiten.
+
+Die W3C-Initiative für barrierefreies Web – die Web Accessibility Initiative ([WAI](/about/participating/)) – entwickelt technische Spezifikationen, Richtlinien, Techniken sowie unterstützende Ressourcen, die Lösungen für Barrierefreiheit beschreiben. Diese gelten als internationale Standards für barrierefreies Web. Die Richtlinien für barrierefreie Webinhalte, <abbr title="Web Content Accessibility Guidelines (WCAG)">WCAG</abbr> 2.0, ist beispielsweise auch ein <abbr title="International Organization for Standardization">ISO</abbr> Standard: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.
```

Comment:
@alexejrotar Would you please add `lang="en"` to each `<abbr>` element?

Example:  `<abbr title="Web Content Accessibility Guidelines (WCAG)" lang="en">WCAG</abbr>`

See https://www.w3.org/International/questions/qa-html-language-declarations.de.html for context